examples/notebooks/hydration_layer.ipynb

```diff
@@ -15,7 +15,7 @@
   },
```

Comment:
<p>Let's not commit new figures as it increases our repo size.</p><p> </p><p>Please commit just this change below:</p><p>N = 1000000&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;# number of points used to sample the diffraction volume</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chuckie82/pysingfel/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>